test/model/wavenet/test_model.py

```diff
@@ -27,21 +27,18 @@ def hyperparameters(dsinfo):
         num_cells=2,
         num_layers=1,
         context_length=2,
-        num_batches_per_epoch=1,
+        num_batches_per_epoch=5,
```

Comment:
Why not always run with `1`?

Reply:
@lostella asked for half of what is currently see the comment below
https://github.com/awslabs/gluon-ts/issues/774#issuecomment-620164912

Reply:
If there are strong feelings about making this test much faster, it's fine for me to lower it further. Maybe just make sure that more than one batch is run, since the network is used differently in the first two forward steps, when hybridizing the network.

Reply:
Does `num_batches_per_epoch=3` sound good then? Like this we both initialise the algorithm and then run one pass after it being initialised so that all parts are properly exercised.

Reply:
`3` sounds alright. Does it pass consistently then?

Reply:
Yes it does, it passes after running 50 iterations hybridised and 50 iterations not hybridised for each dataset. I.e. it passes 200 tests in a row.